modules/EvseManager/manifest.yaml

```diff
@@ -338,6 +338,20 @@ config:
       - UUID_BASE64
       - SHORT_BASE64
     default: UUID
+  supported_operation_modes:
+    description: >-
+      Comma separated value list representing the list of supported operation modes.
+      These are based on the enum values of OperationModeEnum, which are
+      Idle
+      ChargingOnly
+      CentralSetpoint
+      ExternalSetpoint
+      ExternalLimits
+      CentralFrequency
+      LocalFrequency
+      LocalLoadBalancing
+    type: string
+    default: Idle,ChargingOnly
```

Comment:
I'd prefer to drop this config parameter definition for now until we know better how we integrate the supported operation modes. For now it is anyways just a placeholder and not part of any business logic